core/examples/Examples_DiaMatrix.cpp

```diff
@@ -108,7 +108,7 @@ int main() {
     Morpheus::copy(A, B_from_coo);  // should be same with B
     Morpheus::copy(B, A_from_dia);  // should be same with A
 
-    std::cout << "============ Checking build_large ============\n";
+    std::cout << "============ Checking build_large ============" << std::endl;
```

Comment:
sometimes segfaults after build_large case. need to check if we go out of bounds.